src/main/java/cr0s/warpdrive/config/WarpDriveConfig.java

```diff
@@ -263,11 +264,11 @@ public class WarpDriveConfig {
 	public static boolean LOGGING_RENDERING = false;
 	public static boolean LOGGING_CHUNK_HANDLER = false;
 	public static boolean LOGGING_CHUNK_RELOADING = false;
-	public static boolean LOGGING_CHUNK_LOADING = true;
+	public static boolean LOGGING_CHUNK_LOADING = false;
 	public static boolean LOGGING_ENTITY_FX = false;
 	public static boolean LOGGING_CLIENT_SYNCHRONIZATION = false;
 	public static boolean LOGGING_GRAVITY = false;
-	public static boolean LOGGING_OFFLINE_AVATAR = true;
+	public static boolean LOGGING_OFFLINE_AVATAR = false;
```

Comment:
Changing default configurations is outside the scope of your merge request, consider reverting and, optionally, move to another merge request?